chains/81457/token-list.json

```diff
@@ -384,6 +384,13 @@
     "tokenDecimals": 18,
     "tokenCategory": "DeFi"
   },
+  {
+    "tokenAddress": "0xd9a1266a23ed88fc9cb12454a5d6d338b65a3302",
```

Comment:
u did not run the script to autogenerate this file, it should be alphabetically ordered

Reply:
are you running the script from the root directory?